### ISSUE TITLE
How to use TFN for classifications ?

### ISSUE THREAD

**Issue by shamanez** [jdb78/pytorch-forecasting#1045]
The [documentation](https://pytorch-forecasting.readthedocs.io/en/latest/models.html)  TFN can also be use for classification tasks.   How I can proceed with this?

**Comment by misterspook** [comment]
I think that you can pass the categorical variables as a target and define the loss function to be cross entropy.
